wgsl/index.bs

```diff
@@ -560,40 +560,44 @@ type.
 ### Read-only Storage Texture Types ### {#texture-ro}
 <pre class='def'>
 `texture_ro_1d<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 1D 0 0 0 2 image_storage_type ReadOnly
+  %1 = OpTypeImage type(image_storage_type) 1D 0 0 0 2 image_storage_type
 
 `texture_ro_1d_array<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 1D 0 1 0 2 image_storage_type ReadOnly
+  %1 = OpTypeImage type(image_storage_type) 1D 0 1 0 2 image_storage_type
 
 `texture_ro_2d<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 0 0 0 2 image_storage_type ReadOnly
+  %1 = OpTypeImage type(image_storage_type) 0 0 0 2 image_storage_type
 
 `texture_ro_2d_array<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 0 1 0 2 image_storage_type ReadOnly
+  %1 = OpTypeImage type(image_storage_type) 0 1 0 2 image_storage_type
 
 `texture_ro_3d<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 3D 0 0 0 2 image_storage_type ReadOnly
+  %1 = OpTypeImage type(image_storage_type) 3D 0 0 0 2 image_storage_type
 </pre>
 * type(image_storage_type) is one of `f32`, `i32`, or `u32`, depending on the storage type.
 
+When mapping to SPIR-V, a read-only storage texture variable must also have a `NonWritable` decoration.
```

Comment:
Would it be worth showing an example of that?

```
%2 = OpVariable %1 Function
OpDecorate %2 NonWritable
```

Reply:
Done.